tasks/update.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Enable maintenance mode
+  shell: "sudo -u {{ nextcloud_system_user }} php occ maintenance:mode --on"
+  args:
+    chdir: "{{ nextcloud_path }}"
+
+- name: MySQL and Redis
+  systemd:
+    name: "{{ item }}"
+    state: stopped
+  with_items:
+    - nginx
+    - redis
+    - mysql
+
+- name: "Move Nextcloud dir to {{ nextcloud_path_old }}"
```

Comment:
What if `nextcloud_path_old` already exists or contains old data?
Maybe autmatically name the folder depending on the version number.

Reply:
fixed